src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -119,7 +119,7 @@ body {
     ```css title="src/styles/global.css" ins={4}
```

Comment:
``````suggestion
    ```css title="src/styles/global.css" ins={4} ins="inline"
``````
Reuben might have meant highlight as in make the code sample highlight the word inline! If the Tailwind docs suggest this usage, then we don't need any comment about it I don't think. We just tell people to use it! But updating our markdown block to annotate this might be helpful to draw people's attention to it!

Reply:
Oh I see, fancy starlight magic, that makes sense 😄